bitnami/mariadb/10.11/debian-12/rootfs/opt/bitnami/scripts/libmariadb.sh

```diff
@@ -445,7 +445,7 @@ mysql_start_bg() {
     local -a flags=("--defaults-file=${DB_CONF_FILE}" "--basedir=${DB_BASE_DIR}" "--datadir=${DB_DATA_DIR}" "--socket=${DB_SOCKET_FILE}")
 
     # Only allow local connections until MySQL is fully initialized, to avoid apps trying to connect to MySQL before it is fully initialized
-    flags+=("--bind-address=127.0.0.1")
+    flags+=("--skip-networking")
```

Comment:
Please correct me if I'm wrong but according to [MariaDB docs](https://mariadb.com/docs/server/ref/mdb/system-variables/skip_networking) this option will disable any connection via TCP/IP, hence when we start MariaDB in background we won't be able to run commands for operations such as configuring authentication. For instance, operations like the ones below:

- https://github.com/bitnami/containers/blob/main/bitnami/mariadb/10.11/debian-12/rootfs/opt/bitnami/scripts/libmariadb.sh#L344-L351

Am I right?

Reply:
I am not an expert at all, but this option disables tcp/ip networking, I suppose the connection through sockets is working.

Just for reference, official mariadb is using this option during initialization : https://github.com/MariaDB/mariadb-docker/blob/master/docker-entrypoint.sh#L121

And when testing the container with my change, I can connect after initialization with the user created at startup, and my init scripts are executed as well.